scripts/src/indexfile/index.py

```diff
@@ -38,5 +43,97 @@ def get_chart_info(tar_name):
     print(f"[INFO] match not found: {tar_name}")
     return "","","",""
 
+def get_charts_info():
+    chart_info_list = []
+
+    index_dct = _load_index_yaml("https://charts.openshift.io/index.yaml")
```

Comment:
We are using the URL in two functions... It be good to move the URL to a global variable